test/const_eval_test.cpp

```diff
@@ -47,6 +47,10 @@ struct non_computable_cf
 struct test_context
 {
     void finish() const {}
+    void set_stream(int) {}
+    void create_events(int) {}
+    void record_event(int) {}
+    void wait_event(int) {}
```

Comment:
These method dont need to be added here.

Reply:
It was relics of earliest version of context.hpp which has all these methods.

Reply:
fixed.